src/eth1spec/evm.py

```diff
@@ -9,13 +9,22 @@
 from dataclasses import dataclass
 from typing import List, Tuple
 
-from .eth_types import U256, Address, Hash32, Log, State
-from .number import Uint
+from .eth_types import EMPTY_ACCOUNT, U256, Address, Hash32, Log, State, Uint
 
-ADD = b"\x01"
-"""
-Opcode for the addition (`+`) instruction.
-"""
+ops = {
+    "ADD": b"\x01",
+    "PUSH1": b"\x60",
+    "SSTORE": b"\x55",
+}
```

Comment:
I don't think this renders well in the documentation:

![image](https://user-images.githubusercontent.com/57262657/121301792-b7f07100-c8c6-11eb-806d-0ee0aa547323.png)

What if we did something like:

```python
ADD = 0x01
PUSH1 = 0x60
SSTORE = 0x55

ops = {
    "ADD": ADD,
    "PUSH1": PUSH1,
    "SSTORE": SSTORE,
}
```

Which still isn't great, but maybe a bit more useful:

![image](https://user-images.githubusercontent.com/57262657/121302305-81672600-c8c7-11eb-8ce3-4fa0e30beabc.png)

Reply:
Yeah, I'm really unsure about how to lay this portion out. It's critical that it looks nice. I'll go with this for now and play around with it later.